benchmarks/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -402,6 +405,7 @@ int main_(std::vector<std::string> const &args, const MPI_Comm comm)
     // NOTE: minus "1+sqrt(3)/2 \approx 1.37" matches the size of the boxes
     // inserted into the tree (mid-point between half-edge and
     // half-diagonal)
+    // FIXME
```

Comment:
I need help fixing this. I don't remember now how this formula was computed, and we need to adapt it for both 2D and 1D.

Reply:
@dalg24 Do you remember the exact details of computing this? I have trouble reproducing the formula that is in the code.